Source/Common/singleton.cpp

```diff
@@ -75,13 +100,13 @@ std::shared_ptr<http_task_completed_queue> http_singleton::get_task_completed_qu
     return taskQueue;
 }
 
-void verify_http_singleton()
+void verify_http_singleton(_In_ std::shared_ptr<http_singleton>& httpSingleton)
```

Comment:
get_http_singleton() already does the checking, do we really want a second function?

Reply:
Ah, I'll remove